src/components/Forms/Artist/index.tsx

```diff
@@ -1,10 +1,21 @@
 import * as Form from '@radix-ui/react-form'
-import { Input, Toggle } from '../Fields'
+import { Input } from '../Fields'
 import Upload from '../Fields/Upload'
 import { Heading, Flex, Text, Box, Button } from '@radix-ui/themes'
 import useArtistForm from './hooks/useArtistForm'
+import { useRouter } from 'next/router'
+import { api } from '~/utils/api'
+import { useEffect } from 'react'
 
 export default function ArtistForm() {
+    const router = useRouter()
+    const param = router.query.id as string
+    const { data } = api.artist.get.useQuery(
+        { id: param },
+        { enabled: Boolean(param) }
+    )
+    const editing = !!param
```

Comment:
No need to create another variable here. Just pass `Boolean(param)` to the hook or `!!param`